### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by gmerveartar**
Maven and IntelliJ file extensions are added to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,29 @@
...
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+pom.xml.bak
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+.mvn/wrapper/maven-wrapper.jar
```
Comment (ms2892) [inline .gitignore:29]: shouldn't there be a `*.class` as well?

Comment (gmerveartar) [reply .gitignore:29]: yep, I'll add those as well

**Review by ms2892 — APPROVED** [review]
Looks good to me. I love the addition of *.class in gitignore. It did help reduce the number of files significantly

**Review by ms2892 — APPROVED** [review]

File: Libraries/UtilLib/src/main/java/org/csystem/util/numeric/NumberUtil.java [hunk 2]
```diff
@@ -167,6 +186,20 @@ public static long getPrime(int n)
...
 
+    public static BigInteger getPrime(BigInteger n)
+    {
+        var count = BigInteger.ZERO;
+        var val = BigInteger.TWO;
+
+        while (true) {
+            if (isPrime(val))
+                count = count.add(BigInteger.ONE);
+            if (count == n)
```
Comment (ms2892) [inline Libraries/UtilLib/src/main/java/org/csystem/util/numeric/NumberUtil.java]: This comparison doesn't work. You will have to change this line to 

```
if (count.compareTo(n)==0)
```

**Review by ms2892 — CHANGES_REQUESTED** [review]

**Review by ms2892 — APPROVED** [review]
looks good to me

**Pull request merged**